lib/wallet.py

```diff
@@ -1542,9 +1542,20 @@ class Simple_Wallet(Abstract_Wallet):
 
     """ Wallet with a single pubkey per address """
 
+    def decide_txin_type(self):
```

Comment:
Wouldn't a method `get_txin_type` that returns the type be better? The name doesn't really explain what the method does.